lib/src/metta/runner/mod.rs

```diff
@@ -856,6 +870,39 @@ impl<'input> RunContext<'_, '_, 'input> {
         })
     }
 
+    /// Resolves a dependency module from a name, according to the [PkgInfo] of the current module,
+    /// and loads the specified resource from the module, without loading the module itself
+    ///
+    /// NOTE: Although this method won't load the module itself, it will load parent modules if necessary
+    pub fn load_resource_from_module(&mut self, mod_name: &str, res_key: &str) -> Result<Vec<u8>, String> {
+
+        // LP-TODO-NOW!, We should assume mod_name is relative by default, not absolute
+
+        // Resolve the module name and see if the module is already loaded into the runner
+        if let Ok(mod_id) = self.get_module_by_name(mod_name) {
+            self.metta().get_module_resource(mod_id, res_key)
+        } else {
+            #[cfg(not(feature = "pkg_mgmt"))]
```

Comment:
While we have this feature, the code doesn't compile if feature is removed from list of features. Thus I am not sure we need it.